package.json

```diff
@@ -42,8 +43,15 @@
     "eslint": "^8.24.0",
     "eslint-config-cheminfo-typescript": "^11.1.0",
     "jest": "^29.0.3",
+    "jest-matcher-deep-close-to": "^3.0.2",
     "prettier": "^2.7.1",
     "rimraf": "^3.0.2",
     "typescript": "^4.8.3"
+  },
+  "dependencies": {
+    "cheminfo-build": "^1.1.11",
+    "ml-array-max": "^1.2.4",
```

Comment:
Deprecated. We should use the methods of ml-spectra-processing

Reply:
Let me check